client/app/layouts/splash.html

```diff
@@ -21,7 +28,7 @@
   <div layout="column" layout-align="center center" flex="noshrink" layout-margin>
     <div layout layout-align="center center">
       <logo></logo>
-      <h1 class="title">Foodsaving Worldwide</h1>
+      <md-heading-splash align="center">Foodsaving<br>Worldwide</md-heading-splash>
```

Comment:
custom element names are intended for angular directives or components, if its just a normal element then using the most semantic element is best, such as h1